packages/cxl-ui/package.json

```diff
@@ -21,6 +21,7 @@
     "@vaadin/vaadin-progress-bar": "^2.0.0-alpha2",
     "@vaadin/vaadin-tabs": "^4.0.0-alpha4",
     "@vaadin/vaadin-themable-mixin": "^1.6.2",
+    "@vaadin/vaadin-notification": "^1.6.1",
```

Comment:
Go with latest P3 alphas for everything now https://github.com/vaadin/vaadin-notification/releases

Reply:
> Go with latest P3 alphas for everything now https://github.com/vaadin/vaadin-notification/releases

Bumped the version to **2.0.0-alpha1**